### PR TITLE
[blog] add Symfony 3.3 related Symplify deprecations post

### PR DESCRIPTION
**Pull request by TomasVotruba**
@martinahynkova 
Can you check it please?

### CODE REVIEW

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    that were supplemented by few of Symplify packages in Symfony 3.2 and bellow.
+    There is no need - as <a href="https://github.com/Lexinek">Honza Mikes</a> said - to <em>bring the wood to the forest</em>. So they were deprecated.
+    <br><br>
+    I will show <strong>why and what packages were deprecated and how to upgrade your app
+    to use Symfony 3.3 instead</strong>.
+'''
+lang: en
+---
+
+I will give you some insight behind deprecations first. It was not instant decision based on few Symfony blog posts,
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: "some insights" or "the insight" ?
what about "provide" instead of "give"?

Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: an instant decision

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+but community, feedback and intuition inspired long process to mature.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: but  a long process of maturing inspired by community, feedback and intuition

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+If you only want to see before/after changes in you application, you can [skip right to the code](#a-href-https-github-com-deprecatedpackages-defaultautowire-defaultautowire-a). 
+
+<div class="text-center">
+    <img src="/../../../../assets/images/posts/2017/symplify-deprecations/pr-notes.png" style="max-width:100%" class="thumbnail">
+</div>
+
+
+## Bringing The Wood to the Open Source Forest
+
+One of motivations to create and build open-source project is <strong>competition</strong>. You wan't to create better software
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: want

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+than there already is, to push the society further in the development. If my project is the best, there is much less motivation to develop it.
+
+That's why it's the best to have **2 elements of similar level so they push each other forward**, like:
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: it is best to have (without "the")

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+Google & Facebook, USA & Russia, Symfony & Laravel or PHP & Javascript (in a way).
+
+## Swallowing My Open-Source Ego
+ 
+When [Symfony 3.3 introduced new Dependency Injection features](/blog/2017/05/07/how-to-refactor-to-new-dependency-injection-features-in-symfony-3-3/),
+I realized they're 90 % similar to feature I created my packages for.
+ to
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: ?

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+Internal dialog of my Ego and inner Zen master started:
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: has started

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+
+- "It's still not the same, I should keep them and promote them."
+- "But I can do much greater good than adding 10% value. I could focus on completely new packages that bring 100 % value."
+- "But I created them and took care of them for a long time. People should use them."
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: I have created them and I have been taking care of them for a long time

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+- "It's ok, they were good in a time, but things have changed."
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: I am not sure with "in a time"... what about "in the past" ?

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+**So I [asked people on Slack and Github](https://github.com/Symplify/Symplify/pull/162)** what they think about it. Almost everybody (my inner Zen master included)
+agreed **to drop them and let Symfony handle that**.
+ 
+I must say, it was much easier to decide after getting such feedback. **Thank you [Filip Prochazka](https://filip-prochazka.com/),
+[Tomas Fejfar](https://www.tomasfejfar.cz/),  [Stof](https://github.com/Symplify/Symplify/issues/161), 
+[theofidry](https://github.com/symfony/symfony/pull/22234#issuecomment-297999703),
+[Jachym Tousek](https://github.com/enumag), [Jan Mikes](https://github.com/lexinek)
+and [Javier Eguilez](https://github.com/Symplify/Symplify/pull/162#issuecomment-299441503)**
+for help with this.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: for your help

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+
+  
+## Letting Package Go
+
+I realized I could let packages go and made few PRs:
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: "and I made a few PRs" or "I made these few PRs"

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+[Use `_defaults` section](/blog/2017/05/07/how-to-refactor-to-new-dependency-injection-features-in-symfony-3-3/#1-let-s-add-code-defaults-code):
+
+```yaml
+# app/config/services.yml
+services:
+    _defaults:
+        autowire: true
+```
+
+All services in this config will be now autowired. You can use it in any `.yml` file.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: ...autowired now.

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+
+
+## [ServiceDefinitionDecorator](https://github.com/DeprecatedPackages/ServiceDefinitionDecorator)
+
+This packages helped with service tagging and method calls by type.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: These packages

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 13]
```diff
@@ -0,0 +1,178 @@
...
+        resource: ../❴Controller,Command,Subscriber❵
+```
+
+Tada!
+
+And just like that, you can drop 4 of Symplify bundles and use Symfony 3.3 happily and safely ever after.
+
+---
+
+That's it for today. In next post, I will show you, **how to deprecate package without letting anyone behind**.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: That´s "all" for today. I "the" next post I will show you how... (bez čárek stačí)

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+Not even your blog readers that could came across and old post about it.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: ??...can c**o**me across "an" old post ???

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md (hunk 4 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: I think "the" is always used in 3rd form. The longest, the fastest etc.

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md (hunk 5 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: typo, thanks

File: source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md (hunk 8 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-05-18-symfony33-di-symplify-deprecations.md]: Yep, I like that.

**Comment by TomasVotruba** [comment]
Thank you @martinahynkova 
Me like you English a lot. It's easier to read :+1:

**Pull request merged**